spacy_llm/tasks/span/parser.py

```diff
@@ -44,22 +49,22 @@ def parse_responses(responses: Iterable[str], **kwargs) -> Iterable[Span]:
         label_dict (Dict[str, str]): Mapping of normalized to non-normalized labels.
     RETURNS (Iterable[Span]): Parsed spans per doc/response.
     """
-    for doc, prompt_response in zip(kwargs["docs"], responses):
+    for doc, prompt_response in zip(docs, responses):
         spans = []
         for label, phrases in _format_response(
-            prompt_response, kwargs["normalizer"], kwargs["label_dict"]
+            prompt_response, task._normalizer, task._label_dict
         ):
             # For each phrase, find the substrings in the text
             # and create a Span
             offsets = find_substrings(
                 doc.text,
                 phrases,
-                case_sensitive=kwargs["case_sensitive_matching"],
-                single_match=kwargs["single_match"],
+                case_sensitive=task._case_sensitive_matching,
+                single_match=task._single_match,
```

Comment:
We're accessing private vars on the task now so that's the one downside (we'd probably need to make properties for our build-in tasks for these.